packages/core/src/browser/style/scrollbars.css

```diff
@@ -1,6 +1,8 @@
+@import url('~perfect-scrollbar/css/perfect-scrollbar.css');
+
 ::-webkit-scrollbar {
-    height: 9px;
-    width: 9px;
+    height: var(--theia-scrollbar-width);
```

Comment:
is it expected to have width for height ?

Reply:
The variable `theia-scrollbar-width` is used for both vertical and horizontal scrollbars. Is that what you asked about?

Reply:
yes it was looking odd to specify the height of an element from a width variable

Reply:
if height and width are the same value, I would have said that `theia-scrollbar-size` variable was more appropriate. Or to have also `theia-scrollbar-height`variable if someone want to have different values

Reply:
It is not height vs. width, but length vs. width. In that light talking about the width of a scrollbar is pretty clear, no matter it is vertical or horizontal. A variable `size` would be vague, in that it is not clear whether it is the `length` or the `width`.

Reply:
I'm just saying that `width` is not appropriate to set the `height`, `length` is better.

Reply:
IMHO the issue is that `webkit-scrollbar` uses `height` and `width` properties for `vertical-scrollbar-width` and `horizontal-scrollbar-width`. But it also doesn't really matter here, because it is an implementation detail. The exposed 'api' is `theia-scollbar-width` that is one css variable to specify the width of a scrollbar no matter it is vertical or horizontal. 
Does that make sense?